src/main/java/org/opensearch/sdk/ExtensionNamedWriteableRegistry.java

```diff
@@ -105,7 +107,8 @@ public ExtensionBooleanResponse handleNamedWriteableRegistryParseRequest(NamedWr
         boolean status = false;
 
         // Extract data from request and procress fully qualified category class name into class instance
-        Class categoryClass = request.getCategoryClass();
+        @SuppressWarnings("unchecked")
```

Comment:
Huh, so I made the comment earlier thinking this needed it, but the return type of `request.getCategoryClass()` is already  `Class<? extends NamedWriteable>`, so this shouldn't need to be suppressed at all.